### ISSUE TITLE
Provide better way for debug

### ISSUE THREAD

**Issue by tadatuta** [bem/bem-xjst#77]
Now any bug in templates with `bem-xjst 2.x` results with stack trace pointing to
```js
var templates = code.toString().replace(/^function[^{]+{|}$/g, '');
  templates = new Function(BEMHTML.locals.join(', '), templates);
```
which is kinda useless.

**Comment by indutny** [comment]
Yeah... it is terrible. Let me see if I can improve it...

**Comment by indutny** [comment]
Fix is here: 4cc90a0 . It will get better a bit ;)

**Comment by indutny** [comment]
Also, released everything in `3.0.0`, because I broke backwards compatibility while removing `.apply.call(...)`.
